src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -289,14 +282,13 @@ function ReimbursementAccountPage({route, policy}: ReimbursementAccountPageProps
             Navigation.setParams({stepToOpen: getRouteForCurrentStep(currentStep)});
         },
         // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
-        [isOffline, reimbursementAccount, route, hasACHDataBeenLoaded, shouldShowContinueSetupButton],
+        [isOffline, reimbursementAccount, route, hasACHDataBeenLoaded, shouldShowContinueSetupButton, policyIDParam],
     );
 
     const setManualStep = () => {
         BankAccounts.setBankAccountSubStep(CONST.BANK_ACCOUNT.SETUP_TYPE.MANUAL).then(() => {
             setShouldShowContinueSetupButton(false);
         });
-        fetchData(true);
```

Comment:
@koko57 same here, Why do you remove this line?

Reply:
we don't need to fetch the data here. Besides, when I was working on that that call to API seemed not to be sent

Reply:
@koko57 To safe, If we remove this, I think we need to understand why we added it before

Reply:
This flow is quite complicated. So we should be careful about this change to avoid regression. Thanks

Reply:
@DylanDylann as we fetchData when the component mounts we don't need to fetch it once again when we choose manual flow. And as I said - when working on it I've never saw this request being sent (not sure why).

Reply:
@DylanDylann yeah it's complicates 😅 I've tried to simplify it as much as possible.